enterprise/api/v1/serializers.py

```diff
@@ -840,7 +840,17 @@ def validate(self, data):  # pylint: disable=arguments-renamed
         if error:
             raise serializers.ValidationError(error)
 
+        uuid = data.get('uuid')
```

Comment:
the best way to capture this logically would be something like this:
`create_report = data.get('uuid') is None`